glib/src/param_spec.rs

```diff
@@ -1370,6 +1370,77 @@ impl ParamSpecVariant {
     }
 }
 
+pub trait Parametrize {
+    type Spec;
+    fn get(&self) -> crate::Value;
+}
+pub trait ParametrizeMut {
+    fn set(&self, value: &crate::Value);
+}
+
+use crate::ToValue;
+impl Parametrize for String {
+    type Spec = ParamSpecString;
+    fn get(&self) -> crate::Value {
+        self.to_value()
+    }
+}
+impl Parametrize for crate::Object {
+    type Spec = ParamSpecObject;
+    fn get(&self) -> crate::Value {
+        self.to_value()
+    }
+}
+impl Parametrize for f64 {
+    type Spec = ParamSpecDouble;
+    fn get(&self) -> crate::Value {
+        self.to_value()
+    }
+}
+impl Parametrize for f32 {
+    type Spec = ParamSpecFloat;
+    fn get(&self) -> crate::Value {
+        self.to_value()
+    }
+}
+impl Parametrize for i64 {
+    type Spec = ParamSpecInt64;
+    fn get(&self) -> crate::Value {
+        self.to_value()
+    }
+}
+impl Parametrize for i32 {
+    type Spec = ParamSpecInt;
+    fn get(&self) -> crate::Value {
+        self.to_value()
+    }
+}
+impl Parametrize for u64 {
+    type Spec = ParamSpecUInt64;
+    fn get(&self) -> crate::Value {
+        self.to_value()
+    }
+}
+impl Parametrize for u32 {
+    type Spec = ParamSpecUInt;
+    fn get(&self) -> crate::Value {
+        self.to_value()
+    }
+}
+impl<T: Parametrize + ToValue> Parametrize for std::cell::RefCell<T> {
+    type Spec = T::Spec;
+    fn get(&self) -> crate::Value {
+        self.borrow().to_value()
+    }
+}
+impl<T: 'static + Parametrize + for <'b> crate::value::FromValue<'b>> ParametrizeMut for std::cell::RefCell<T> {
```

Comment:
You will need similar implementations for Cell/Mutex.

Reply:
And `RWLock` and `Box`

Reply:
Can't implement ParametrizeMut on Box since it's not mutable. I'll do it for the other ones